.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
Please add gitignore rule for redactor configuration

Reply:
Added